scripts/run-tox-scenario

```diff
@@ -5,4 +5,4 @@ shift
 
 # CircleCI has a bug in its workspace code where it can't handle filenames with some chars
 CLEANED_PATTERN=`echo $PATTERN | tr '^?()$' '_'`
-exec tox -l | grep "$PATTERN" | tr '\n' ',' | xargs -I ARGS tox --result-json /tmp/"$CLEANED_PATTERN".results -e ARGS -- $@
+exec tox -l | grep "$PATTERN" | tr '\n' ',' | xargs -I ARGS tox -e ARGS -- $@
```

Comment:
we should do this in a different PR, unless it is related here?

Reply:
sure, i think it was supposed to be anyway but it never happened upstream